examples/mission-control/src/mission_control.rs

```diff
@@ -87,35 +87,11 @@ fn rates_default() -> HashMap<Exchange, Rate> {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use near_sdk::MockedBlockchain;
-    use near_sdk::{testing_env, VMContext};
-
-    fn get_context(input: Vec<u8>, is_view: bool) -> VMContext {
```

Comment:
So by having a default mocked blockchain, we don't have to go thru the shenanigans of setting up the context here and also explicitly using `testing_env!` right? This is pretty nice for setting up new tests 👍.

Reply:
yeah, you still need it if you need to configure anything, but if not this saves some boilerplate.

Ideally, I think it might be nice to just be able to modify the context in place rather than replacing it, but there are a lot of connected pieces rn.